src/beamlime/handlers/to_nx_log.py

```diff
@@ -0,0 +1,87 @@
+from typing import Any
+
+import numpy as np
+import scipp as sc
+from scippnexus.field import _as_datetime as snx_as_datetime
+
+from beamlime.core.handler import Accumulator
+from beamlime.handlers.accumulators import LogData
+
+
+class ToNXlog(Accumulator[LogData, sc.DataArray]):
+    """
+    Preprocessor for log data.
+
+    Accumulates LogData objects and returns a single DataArray as it would be read from
+    an NXlog in a NeXus file. The DataArray grows as data is added and is not cleared
+    until explicitly requested.
```

Comment:
Typically in which cases and by whome will it be clear...?

Reply:
The main handler implementation we are using (`PeriodicAccumulatingHandler`) will clear accumulators when the user presses a button.

Reply:
Ah so by the handlers. Okay.